### PR TITLE
add EyeLogic eye tracking devices to manual

### PR DESCRIPTION
**Pull request by Kumortas**

### CODE REVIEW

File: content/pages/manual/eyetracking/eyelogic.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[TOC]
+
+## About EyeLogic
+
+The eye tracking devices from EyeLogic meet the highest standards in accuracy and precision, required for scientific usage. They excel in providing a low and consistent latency, offer strong robustness as well as a high tolerance of head motion. The exceptionality however lies in the combination of quality criteria in connection with portability and ease of use.
+
+- <https://www.eyelogic.de/>
+
+The EyeLogic OpenSesame-plugin is included in the official release of OpenSesame, versions from 3.3.10.
```
Comment (smathot) [inline content/pages/manual/eyetracking/eyelogic.md]: I assume you mean that EyeLogic is now supported by the PyGaze plugins, right? (Not that there's a separate plug-in.)

Comment (Kumortas) [reply content/pages/manual/eyetracking/eyelogic.md]: Yes, I clariefied this sentence (plz see new pull-request).

**Comment by smathot** [comment]
Thanks again for the PR and your work on the PyGaze implementation. I did not merge this PR because the text was a little too advertisement-like for my taste. However, I did include an EyeLogic entry in the docs.

**Comment by Kumortas** [comment]
Oh, I am sad to hear that. I tried to do the EyeLogic entry similar to the other entries of eye trackers: Just entries to the menu and a short howto. I am not sure what you mean by advertisement-like. I am open for suggestions on any text changes.

Anyway, thank you for your support and for including EyeLogic into PyGaze. I would be happy if there is a possibility to add a short howto for EyeLogic.

**Comment by smathot** [comment]
> I am not sure what you mean by advertisement-like. 

Well … ;-) 

> The eye tracking devices from EyeLogic meet the highest standards in accuracy and precision, required for scientific usage. They excel in providing a low and consistent latency, offer strong robustness as well as a high tolerance of head motion. The exceptionality however lies in the combination of quality criteria in connection with portability and ease of use.

Regarding this though:

> I would be happy if there is a possibility to add a short howto for EyeLogic.

In principle I'm ok with that, but I have the impression that it's just a matter of starting the server and launching the experiment, which doesn't really require a howto. Nevertheless, if you want to expand on the page that's ok. Just keep it factual!

**Comment by Kumortas** [comment]
You mean that introduction sentence. For those who don't know the company, aimed to be a short intro. You are right, it is formulated adverticement-like, it is just a default intro. I have seen that other trackers have a similar short intro. But anyway, its not too important. So I am fine with as it is.

**Pull request closed without merging**